ariff.html

```diff
@@ -160,7 +160,13 @@ <h2>Co-curricular Activities</h2>
             <div class="entry d-flex align-items-center">
               <div class="detail py-3">
                 <p><strong>1) Softball</strong></p>
+                <ul>
+                  <li>Softball Nationals 2021 - I participated in the Softball Nationals </li>
+                </ul>
                 <p><strong>2) Students' Council</strong></p>
+                <p>1) Handbook - I designed the handbook for 2021
```

Comment:
Remember to change to bullet points so it looks nicer (ul and li, can copy above)